slidedeck/package.json

```diff
@@ -9,37 +9,49 @@
     "Mike Hostetler",
     "Justin Perinovic",
     "Jeff Sheets",
-    "Travis Martensen"
+    "Travis Martensen",
+    "Dustin Schau <dustin.schau@objectpartners.com>"
   ],
   "license": "MIT",
   "scripts": {
+    "precommit": "pretty-quick --staged",
     "prebuild": "del-cli \"dist/**/*\"",
-    "build": "webpack --env.production",
-    "predeploy": "npm run build",
-    "deploy": "ghpages --path dist",
-    "start": "webpack-dev-server --env.development --inline --progress --port 9998 --content-base ./src/public --content-base ./",
-    "test": "echo \"We should probably write some tests huh\""
+    "build": "cross-env NODE_ENV=production webpack --env.environment",
+    "start": "cross-env NODE_ENV=development webpack-dev-server",
+    "test": "echo \"We should probably write some tests\""
   },
   "devDependencies": {
-    "@objectpartners/presentation-config": "^1.0.0",
-    "@types/node": "~7.0.12",
-    "@types/react": "~15.0.23",
-    "@types/react-dom": "~15.5.0",
-    "@types/webpack-env": "~1.13.0",
-    "copy-webpack-plugin": "~4.0.1",
+    "autoprefixer": "~8.3.0",
+    "babel": "~6.23.0",
+    "babel-loader": "~7.1.4",
+    "babel-preset-env": "~1.6.1",
+    "babel-preset-react": "~6.24.1",
+    "cross-env": "~5.1.4",
+    "css-loader": "~0.28.11",
+    "cssnano": "~3.10.0",
     "del-cli": "~0.2.1",
-    "ghpages": "~0.0.10",
-    "object-partners-presentation": "objectpartners/object-partners-presentation-config#1.0.1",
```

Comment:
Is this not published on `npm` like before?

Reply:
I published it on NPM the whole time, we were just using the Github tag. I think I prefer using the variant on NPM.